readme.md

```diff
@@ -1,78 +1,36 @@
+
 <p align="right">
 <a href="https://autorelease.general.dmz.palantir.tech/palantir/spark-tpcds-benchmark"><img src="https://img.shields.io/badge/Perform%20an-Autorelease-success.svg" alt="Autorelease"></a>
 </p>
 
-TPCDS Spark Benchmark Runner
-==================================
+Spark Benchmark Runner
+======================
+
+This repo contains tools to run 2 industry standard benchmark suites:
 
-TPC-DS is an open benchmark suite for structured data systems. This utility aims to make it easy to generate TPC-DS
-data, and to run TPC-DS benchmarks against different versions of Spark. The main use case for this utility is to run
-scale tests when developing Spark itself. For example, one can use this tool to check the performance of a change made
-to the way Spark shuffle works.
+1. [TPC-DS](http://www.tpc.org/tpcds/) is an open benchmark suite for structured data systems. This utility aims to make it easy to generate TPC-DS data, and to run TPC-DS benchmarks against different versions of Spark. The main use case for this utility is to run scale tests when developing Spark itself. For example, one can use this tool to check the performance of a change made to the way Spark shuffle works. The benchmarks run SQL queries against structured datasets. This utility is thus not useful for running tests in streaming workflows.
```

Comment:
"The main use case for this utility is to run scale tests when developing Spark itself" could maybe be rephrased to "The main use case for this utility is to test performance at scale when evaluating changes to Spark or to its underlying infrastructure."